2020-01-15 18:34:13,598 [MainThread:2164] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:34:13,602 [MainThread:2164] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:34:13,604 [MainThread:2164] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 18:34:13,607 [MainThread:2164] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:34:25,790 [Thread-1:9424] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 8893
2020-01-15 18:34:25,813 [Thread-2:9820] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/css/bootstrap.min.css%20 HTTP/1.1" 304 0
2020-01-15 18:34:25,814 [Thread-1:9424] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/dashboard.css HTTP/1.1" 304 0
2020-01-15 18:34:25,821 [Thread-2:9820] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/popper.js-1.16.0/popper.min.js HTTP/1.1" 304 0
2020-01-15 18:34:25,822 [Thread-1:9424] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/Font-Awesome-5.6.3/css/all.css%20 HTTP/1.1" 304 0
2020-01-15 18:34:25,827 [Thread-2:9820] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/js/bootstrap.min.js HTTP/1.1" 304 0
2020-01-15 18:34:25,859 [Thread-2:9820] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/js/bootstrap.min.js.map HTTP/1.1" 304 0
2020-01-15 18:34:25,870 [Thread-2:9820] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/Font-Awesome-5.6.3/webfonts/fa-solid-900.woff2 HTTP/1.1" 304 0
2020-01-15 18:34:25,884 [Thread-2:9820] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/css/bootstrap.min.css.map HTTP/1.1" 304 0
2020-01-15 18:34:25,943 [Thread-1:9424] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:34:25,944 [Thread-2:9820] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:34:25,944 [Thread-2:9820] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:34:25,951 [Thread-2:9820] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.006) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:34:25,973 [Thread-2:9820] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130577
2020-01-15 18:37:43,400 [MainThread:10444] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:37:43,401 [MainThread:10444] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:37:43,403 [MainThread:10444] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 18:37:43,406 [MainThread:10444] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:37:47,139 [Thread-1:7576] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 8897
2020-01-15 18:37:47,161 [Thread-1:7576] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/popper.js-1.16.0/umd/popper.min.js HTTP/1.1" 200 21262
2020-01-15 18:37:47,204 [Thread-1:7576] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/popper.js-1.16.0/umd/popper.min.js.map HTTP/1.1" 200 123890
2020-01-15 18:37:47,257 [Thread-1:7576] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:37:47,258 [Thread-1:7576] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:37:47,258 [Thread-2:10876] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:37:47,265 [Thread-1:7576] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.007) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:37:47,291 [Thread-1:7576] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130577
2020-01-15 18:38:40,263 [MainThread:9424] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:38:40,263 [MainThread:9424] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:38:40,266 [MainThread:9424] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 18:38:40,269 [MainThread:9424] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:38:45,189 [Thread-1:10772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 8899
2020-01-15 18:38:45,309 [Thread-1:10772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:38:45,310 [Thread-1:10772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:38:45,310 [Thread-2:9800] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:38:45,317 [Thread-1:10772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.007) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:38:45,339 [Thread-1:10772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130577
2020-01-15 18:39:27,693 [MainThread:10084] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:39:27,693 [MainThread:10084] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:39:27,696 [MainThread:10084] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.003) SHOW FULL TABLES; args=None
2020-01-15 18:39:27,699 [MainThread:10084] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:39:36,538 [Thread-1:7612] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py:_resolve_lookup:872] [DEBUG]- Exception while resolving variable 'employeeId' in template 'app04/bootstrapTable01.html'.
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 829, in _resolve_lookup
    current = current[bit]
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'employeeId'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 835, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'employeeId'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 843, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'employeeId'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 850, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [employeeId] in [{'True': True, 'False': False, 'None': None}, {'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x0000000004395438>>, 'request': <WSGIRequest: GET '/app04/table01/'>, 'user': <SimpleLazyObject: <function AuthenticationMiddleware.process_request.<locals>.<lambda> at 0x00000000042080D8>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x00000000043B0448>, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x00000000041EA548>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'SUCCESS': 25, 'WARNING': 30, 'ERROR': 40}}, {}, {'view': <django.views.generic.base.TemplateView object at 0x00000000041ECD48>}, {'block': <Block Node: myMain. Contents: [<TextNode: '\n    '>, <TextNode: '\n    <div id="toolbar">\n '>, <TextNode: '\n    <table\n            i'>, <TextNode: '\n            data-search='>, <TextNode: '\n            '>, <TextNode: '\n            '>, <TextNode: '\n            data-url="'>, <django.template.defaulttags.URLNode object at 0x00000000043ADF08>, <TextNode: '"\n'>, <TextNode: '\n    >\n    </table>\n\n    '>, <TextNode: '\n\n        function getIdS'>, <TextNode: '\n\n        '>, <TextNode: '\n        '>, <TextNode: '\n                '>, <TextNode: '\n        '>, <TextNode: '\n                '>, <TextNode: '\n                '>, <TextNode: '\n                '>, <TextNode: '\n        '>, <TextNode: '\n        '>, <TextNode: '\n        '>, <TextNode: '\n\n        '>, <TextNode: '\n\n        function detail'>, <TextNode: '\n\n        function operat'>, <django.template.defaulttags.CommentNode object at 0x00000000043AADC8>, <TextNode: '\n                '>, <TextNode: '\n                console.'>, <django.template.defaulttags.URLNode object at 0x00000000043AA808>, <TextNode: '")\n\n\n                '>, <django.template.defaulttags.CommentNode object at 0x00000000043AD388>, <TextNode: '\n            }\n        }\n'>, <TextNode: '\n\n        function initTa'>, <TextNode: '\n                '>, <TextNode: '\n                columns:'>, <TextNode: "\n            $table.on('c">, <TextNode: '\n        $(function () {\n'>]>}]
2020-01-15 18:39:36,596 [Thread-1:7612] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\utils\log.py:log_response:228] [ERROR]- Internal Server Error: /app04/table01/
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\handlers\base.py", line 145, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\handlers\base.py", line 143, in _get_response
    response = response.render()
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\response.py", line 106, in render
    self.content = self.rendered_content
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\response.py", line 83, in rendered_content
    content = template.render(context, self._request)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 937, in render
    bit = node.render_annotated(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 904, in render_annotated
    return self.render(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 937, in render
    bit = node.render_annotated(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 904, in render_annotated
    return self.render(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 937, in render
    bit = node.render_annotated(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\base.py", line 904, in render_annotated
    return self.render(context)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\template\defaulttags.py", line 443, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\urls\base.py", line 90, in reverse
    return iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs))
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\urls\resolvers.py", line 660, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'deleteEmployee' with arguments '('',)' not found. 1 pattern(s) tried: ['app04/delEmployee/(?P<employeeID>[0-9]+)/$']
2020-01-15 18:39:36,599 [Thread-1:7612] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [ERROR]- "GET /app04/table01/ HTTP/1.1" 500 169191
2020-01-15 18:39:36,752 [Thread-1:7612] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /favicon.ico HTTP/1.1" 302 0
2020-01-15 18:39:36,760 [Thread-1:7612] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:39:50,151 [MainThread:9736] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:39:50,151 [MainThread:9736] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:39:50,154 [MainThread:9736] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 18:39:50,157 [MainThread:9736] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:39:54,323 [MainThread:8744] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:39:54,323 [MainThread:8744] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:39:54,326 [MainThread:8744] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.003) SHOW FULL TABLES; args=None
2020-01-15 18:39:54,329 [MainThread:8744] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:40:00,090 [Thread-1:9292] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 8972
2020-01-15 18:40:00,255 [Thread-1:9292] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:40:00,256 [Thread-1:9292] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:40:00,264 [Thread-1:9292] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.008) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:40:00,286 [Thread-1:9292] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130577
2020-01-15 18:40:00,289 [Thread-2:8224] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:46:30,962 [MainThread:9956] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:46:30,963 [MainThread:9956] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:46:30,965 [MainThread:9956] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 18:46:30,968 [MainThread:9956] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:46:34,920 [Thread-1:11096] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 8889
2020-01-15 18:46:34,960 [Thread-1:11096] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/dashboard.css HTTP/1.1" 304 0
2020-01-15 18:46:34,961 [Thread-2:4692] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/css/bootstrap.min.css%20 HTTP/1.1" 304 0
2020-01-15 18:46:34,966 [Thread-1:11096] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/js/bootstrap.min.js HTTP/1.1" 304 0
2020-01-15 18:46:34,968 [Thread-2:4692] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/Font-Awesome-5.6.3/css/all.css%20 HTTP/1.1" 304 0
2020-01-15 18:46:35,014 [Thread-2:4692] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/js/bootstrap.min.js.map HTTP/1.1" 304 0
2020-01-15 18:46:35,015 [Thread-1:11096] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/Font-Awesome-5.6.3/webfonts/fa-solid-900.woff2 HTTP/1.1" 304 0
2020-01-15 18:46:35,032 [Thread-1:11096] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/css/bootstrap.min.css.map HTTP/1.1" 304 0
2020-01-15 18:46:35,072 [Thread-1:11096] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:46:35,073 [Thread-1:11096] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:46:35,076 [Thread-2:4692] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:46:35,080 [Thread-1:11096] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.006) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:46:35,100 [Thread-1:11096] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130577
2020-01-15 18:47:59,940 [MainThread:10488] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:47:59,940 [MainThread:10488] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:47:59,943 [MainThread:10488] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 18:47:59,946 [MainThread:10488] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:48:53,458 [MainThread:10864] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:48:53,458 [MainThread:10864] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:48:53,461 [MainThread:10864] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.003) SHOW FULL TABLES; args=None
2020-01-15 18:48:53,464 [MainThread:10864] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:48:57,619 [Thread-1:7788] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 9200
2020-01-15 18:48:57,642 [Thread-1:7788] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/popper.js-1.16.0/umd/popper.min.js HTTP/1.1" 304 0
2020-01-15 18:48:57,692 [Thread-1:7788] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/popper.js-1.16.0/umd/popper.min.js.map HTTP/1.1" 304 0
2020-01-15 18:48:57,742 [Thread-1:7788] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:48:57,743 [Thread-1:7788] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:48:57,753 [Thread-1:7788] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.009) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:48:57,769 [Thread-2:4212] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:48:57,776 [Thread-1:7788] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130577
2020-01-15 18:48:59,958 [Thread-1:7788] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\utils\log.py:log_response:228] [WARNING]- Forbidden (CSRF token missing or incorrect.): /app04/delEmployee/
2020-01-15 18:48:59,960 [Thread-1:7788] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [WARNING]- "DELETE /app04/delEmployee/ HTTP/1.1" 403 2513
2020-01-15 18:49:45,485 [MainThread:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:49:45,485 [MainThread:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:49:45,488 [MainThread:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.003) SHOW FULL TABLES; args=None
2020-01-15 18:49:45,491 [MainThread:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:49:47,656 [Thread-1:9700] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 9186
2020-01-15 18:49:47,775 [Thread-1:9700] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:49:47,776 [Thread-1:9700] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:49:47,785 [Thread-1:9700] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.009) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:49:47,807 [Thread-1:9700] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130577
2020-01-15 18:49:47,807 [Thread-2:10348] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:49:50,232 [Thread-1:9700] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:49:50,232 [Thread-1:9700] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee` WHERE `app04_employee`.`id` = 0; args=(0,)
2020-01-15 18:49:50,240 [Thread-1:9700] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`  LIMIT 21; args=()
2020-01-15 18:49:50,254 [Thread-1:9700] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\utils\log.py:log_response:228] [ERROR]- Internal Server Error: /app04/delEmployee/
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\pythonProject\myDjango00\app04\views.py", line 21, in deletEmployee
    models.Employee.objects.get(id=employeeID).delete()
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\models\query.py", line 408, in get
    self.model._meta.object_name
app04.models.Employee.DoesNotExist: Employee matching query does not exist.
2020-01-15 18:49:50,256 [Thread-1:9700] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [ERROR]- "GET /app04/delEmployee/ HTTP/1.1" 500 18687
2020-01-15 18:51:33,084 [MainThread:9884] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:51:33,085 [MainThread:9884] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:51:33,087 [MainThread:9884] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 18:51:33,090 [MainThread:9884] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:51:35,605 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 9177
2020-01-15 18:51:35,691 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:51:35,730 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:51:35,731 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:51:35,737 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.005) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:51:35,758 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130577
2020-01-15 18:51:38,741 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:51:38,741 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee` WHERE `app04_employee`.`id` = 180; args=(180,)
2020-01-15 18:51:39,196 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.453) DELETE FROM `app04_employee` WHERE `app04_employee`.`id` IN (180); args=(180,)
2020-01-15 18:51:39,204 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/delEmployee/?employeeID=180 HTTP/1.1" 200 12
2020-01-15 18:52:00,378 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:52:00,383 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.004) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:52:00,407 [Thread-1:10900] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130512
2020-01-15 18:53:03,825 [MainThread:10024] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:53:03,825 [MainThread:10024] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:53:03,828 [MainThread:10024] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.003) SHOW FULL TABLES; args=None
2020-01-15 18:53:03,831 [MainThread:10024] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:53:08,055 [Thread-1:10968] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 9177
2020-01-15 18:53:08,147 [Thread-1:10968] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:53:08,181 [Thread-1:10968] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:53:08,182 [Thread-1:10968] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:53:08,188 [Thread-1:10968] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.006) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:53:08,209 [Thread-1:10968] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130512
2020-01-15 18:53:13,034 [Thread-1:10968] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:53:13,038 [Thread-1:10968] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.003) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee` WHERE `app04_employee`.`id` = 2365; args=(2365,)
2020-01-15 18:53:13,553 [Thread-1:10968] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.513) DELETE FROM `app04_employee` WHERE `app04_employee`.`id` IN (2365); args=(2365,)
2020-01-15 18:53:13,602 [Thread-1:10968] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\utils\log.py:log_response:228] [ERROR]- Internal Server Error: /app04/delEmployee/
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\pythonProject\myDjango00\app04\views.py", line 22, in deletEmployee
    return JsonResponse(content={"deleteResult": "ok"})
TypeError: __init__() missing 1 required positional argument: 'data'
2020-01-15 18:53:13,604 [Thread-1:10968] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [ERROR]- "GET /app04/delEmployee/?employeeID=2365 HTTP/1.1" 500 18406
2020-01-15 18:54:01,181 [MainThread:9244] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:54:01,182 [MainThread:9244] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:54:01,184 [MainThread:9244] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 18:54:01,187 [MainThread:9244] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:54:04,363 [Thread-1:4340] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 9177
2020-01-15 18:54:04,443 [Thread-1:4340] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:54:04,481 [Thread-1:4340] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:54:04,482 [Thread-1:4340] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:54:04,490 [Thread-1:4340] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.008) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:54:04,517 [Thread-1:4340] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130446
2020-01-15 18:54:07,224 [Thread-1:4340] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:54:07,224 [Thread-1:4340] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee` WHERE `app04_employee`.`id` = 109; args=(109,)
2020-01-15 18:54:07,739 [Thread-1:4340] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.514) DELETE FROM `app04_employee` WHERE `app04_employee`.`id` IN (109); args=(109,)
2020-01-15 18:54:07,740 [Thread-1:4340] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/delEmployee/?employeeID=109 HTTP/1.1" 200 22
2020-01-15 18:54:47,560 [MainThread:10228] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:54:47,560 [MainThread:10228] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:54:47,563 [MainThread:10228] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.003) SHOW FULL TABLES; args=None
2020-01-15 18:54:47,566 [MainThread:10228] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:55:05,772 [MainThread:10768] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:55:05,773 [MainThread:10768] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:55:05,775 [MainThread:10768] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 18:55:05,778 [MainThread:10768] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:55:08,729 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 9054
2020-01-15 18:55:08,813 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:55:08,847 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:55:08,847 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:55:08,853 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.006) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:55:08,878 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130381
2020-01-15 18:55:16,231 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:55:16,232 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee` WHERE `app04_employee`.`id` = 2797; args=(2797,)
2020-01-15 18:55:16,482 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.249) DELETE FROM `app04_employee` WHERE `app04_employee`.`id` IN (2797); args=(2797,)
2020-01-15 18:55:16,483 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/delEmployee/?employeeID=2797 HTTP/1.1" 200 22
2020-01-15 18:55:36,636 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:55:36,642 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.005) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:55:36,663 [Thread-1:9200] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130315
2020-01-15 18:55:52,265 [MainThread:9676] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:55:52,266 [MainThread:9676] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:55:52,268 [MainThread:9676] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 18:55:52,271 [MainThread:9676] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 18:55:55,815 [Thread-1:3644] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 9177
2020-01-15 18:55:55,906 [Thread-1:3644] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 18:55:55,942 [Thread-1:3644] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 18:55:55,942 [Thread-1:3644] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:55:55,949 [Thread-1:3644] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.007) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 18:55:55,975 [Thread-1:3644] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130315
2020-01-15 18:55:58,868 [Thread-1:3644] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 18:55:58,869 [Thread-1:3644] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee` WHERE `app04_employee`.`id` = 2850; args=(2850,)
2020-01-15 18:55:59,419 [Thread-1:3644] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.549) DELETE FROM `app04_employee` WHERE `app04_employee`.`id` IN (2850); args=(2850,)
2020-01-15 18:55:59,420 [Thread-1:3644] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/delEmployee/?employeeID=2850 HTTP/1.1" 200 22
2020-01-15 19:39:37,056 [MainThread:11428] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 19:39:37,057 [MainThread:11428] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:39:37,059 [MainThread:11428] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 19:39:37,062 [MainThread:11428] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 19:40:03,950 [Thread-1:12072] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 10930
2020-01-15 19:40:03,972 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/dashboard.css HTTP/1.1" 304 0
2020-01-15 19:40:03,973 [Thread-1:12072] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/sweetalert/sweetalert.css%20 HTTP/1.1" 200 13051
2020-01-15 19:40:03,976 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/css/bootstrap.min.css%20 HTTP/1.1" 304 0
2020-01-15 19:40:03,981 [Thread-1:12072] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/Font-Awesome-5.6.3/css/all.css%20 HTTP/1.1" 304 0
2020-01-15 19:40:03,983 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/sweetalert/sweetalert.min.js HTTP/1.1" 200 16651
2020-01-15 19:40:03,987 [Thread-1:12072] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/popper.js-1.16.0/umd/popper.min.js HTTP/1.1" 304 0
2020-01-15 19:40:03,994 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/js/bootstrap.min.js HTTP/1.1" 304 0
2020-01-15 19:40:04,047 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/popper.js-1.16.0/umd/popper.min.js.map HTTP/1.1" 304 0
2020-01-15 19:40:04,048 [Thread-1:12072] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/js/bootstrap.min.js.map HTTP/1.1" 304 0
2020-01-15 19:40:04,052 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/Font-Awesome-5.6.3/webfonts/fa-solid-900.woff2 HTTP/1.1" 304 0
2020-01-15 19:40:04,079 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/bootstrap4.3.1/css/bootstrap.min.css.map HTTP/1.1" 304 0
2020-01-15 19:40:04,117 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 19:40:04,119 [Thread-1:12072] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 19:40:04,119 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:40:04,125 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.005) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 19:40:04,148 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130249
2020-01-15 19:40:26,293 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:40:26,294 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee` WHERE `app04_employee`.`id` = 2793; args=(2793,)
2020-01-15 19:40:26,919 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.623) DELETE FROM `app04_employee` WHERE `app04_employee`.`id` IN (2793); args=(2793,)
2020-01-15 19:40:26,920 [Thread-2:9772] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/delEmployee/?employeeID=2793 HTTP/1.1" 200 22
2020-01-15 19:41:05,986 [MainThread:7432] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 19:41:05,986 [MainThread:7432] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:41:05,988 [MainThread:7432] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SHOW FULL TABLES; args=None
2020-01-15 19:41:05,991 [MainThread:7432] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 19:41:10,112 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 10969
2020-01-15 19:41:10,251 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 19:41:10,254 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:41:10,256 [Thread-2:9732] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 19:41:10,262 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.008) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 19:41:10,284 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130183
2020-01-15 19:41:13,167 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:41:13,168 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee` WHERE `app04_employee`.`id` = 2385; args=(2385,)
2020-01-15 19:41:13,443 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.273) DELETE FROM `app04_employee` WHERE `app04_employee`.`id` IN (2385); args=(2385,)
2020-01-15 19:41:13,444 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/delEmployee/?employeeID=2385 HTTP/1.1" 200 22
2020-01-15 19:41:23,829 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:41:23,830 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee` WHERE `app04_employee`.`id` = 156; args=(156,)
2020-01-15 19:41:24,205 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.375) DELETE FROM `app04_employee` WHERE `app04_employee`.`id` IN (156); args=(156,)
2020-01-15 19:41:24,206 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/delEmployee/?employeeID=156 HTTP/1.1" 200 22
2020-01-15 19:41:59,485 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 10969
2020-01-15 19:41:59,627 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:41:59,628 [Thread-2:9732] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 19:41:59,635 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.008) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 19:41:59,656 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 130052
2020-01-15 19:42:01,979 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:42:01,980 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee` WHERE `app04_employee`.`id` = 435; args=(435,)
2020-01-15 19:42:02,207 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.226) DELETE FROM `app04_employee` WHERE `app04_employee`.`id` IN (435); args=(435,)
2020-01-15 19:42:02,208 [Thread-1:9384] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/delEmployee/?employeeID=435 HTTP/1.1" 200 22
2020-01-15 19:42:31,558 [MainThread:11876] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 19:42:31,558 [MainThread:11876] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:42:31,561 [MainThread:11876] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.003) SHOW FULL TABLES; args=None
2020-01-15 19:42:31,564 [MainThread:11876] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2020-01-15 19:42:34,755 [Thread-1:8732] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/table01/ HTTP/1.1" 200 10971
2020-01-15 19:42:34,874 [Thread-1:8732] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 19:42:34,875 [Thread-1:8732] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:42:34,876 [Thread-2:7248] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /static/images/favicon.ico HTTP/1.1" 200 1150
2020-01-15 19:42:34,881 [Thread-1:8732] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.006) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee`; args=()
2020-01-15 19:42:34,901 [Thread-1:8732] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/getAllEmployees/?search=&sort=name&order=desc HTTP/1.1" 200 129987
2020-01-15 19:42:37,002 [Thread-1:8732] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:42:37,003 [Thread-1:8732] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.001) SELECT `app04_employee`.`id`, `app04_employee`.`name`, `app04_employee`.`age`, `app04_employee`.`address` FROM `app04_employee` WHERE `app04_employee`.`id` = 2411; args=(2411,)
2020-01-15 19:42:37,165 [Thread-1:8732] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.161) DELETE FROM `app04_employee` WHERE `app04_employee`.`id` IN (2411); args=(2411,)
2020-01-15 19:42:37,166 [Thread-1:8732] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\core\servers\basehttp.py:log_message:154] [INFO]- "GET /app04/delEmployee/?employeeID=2411 HTTP/1.1" 200 22
2020-01-15 19:47:39,092 [MainThread:10800] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 19:47:39,093 [MainThread:10800] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:47:39,095 [MainThread:10800] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SELECT `app02_publisher`.`id`, `app02_publisher`.`name`, `app02_publisher`.`addr` FROM `app02_publisher`; args=()
2020-01-15 19:49:19,902 [MainThread:11680] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 19:49:19,903 [MainThread:11680] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:49:19,903 [MainThread:11680] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `app02_publisher`.`id`, `app02_publisher`.`name`, `app02_publisher`.`addr` FROM `app02_publisher`; args=()
2020-01-15 19:49:36,917 [MainThread:9828] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 19:49:36,918 [MainThread:9828] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:49:36,918 [MainThread:9828] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT `app02_publisher`.`id`, `app02_publisher`.`name`, `app02_publisher`.`addr` FROM `app02_publisher`; args=()
2020-01-15 19:51:26,706 [MainThread:11404] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2020-01-15 19:51:26,706 [MainThread:11404] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.000) SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED; args=None
2020-01-15 19:51:26,708 [MainThread:11404] [C:\ProgramData\Anaconda3\envs\envDjango2.2\lib\site-packages\django\db\backends\utils.py:execute:110] [DEBUG]- (0.002) SELECT `app02_publisher`.`id`, `app02_publisher`.`name`, `app02_publisher`.`addr` FROM `app02_publisher`; args=()
